libsplinter/src/biome/key_management/store/diesel/operations/insert_key.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018-2020 Cargill Incorporated
```

Comment:
typo in commit message diesel::operatoins::keys -> diesel::operations::keys

Reply:
Addressed